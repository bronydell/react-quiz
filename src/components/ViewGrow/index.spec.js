import React from 'react'
import { shallow } from 'enzyme'

import Component from './'

it('is rendered', () => {
  shallow(pug`Component`)
})

it('renders children', () => {
  const rendered = shallow(pug`Component Hello World`)

  expect(rendered).toMatchSnapshot()
})
