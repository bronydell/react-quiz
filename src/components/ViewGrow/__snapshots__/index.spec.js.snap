// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders children 1`] = `
<Styled(View)>
  Hello World
</Styled(View)>
`;
